pkg/spiffs/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME=spiffs
+PKG_URL=https://github.com/pellepl/spiffs.git
+PKG_VERSION=39937743fbbec4b82308ee08332bf9180408d23b
```

Comment:
This is the last release (v0.3.6) I'm not sure we should use a random commit from the master branch. I'd say we stick to this one.